### PR TITLE
GUACAMOLE-1503: Japanese Character input cannot be switched By full-width half-width key.

### PR DESCRIPTION
**Pull request by nw-engineer**
Fixed so that full-width and half-width keys work correctly and Japanese input switching works.
Hope to include it in the next release. Thank you.

### CODE REVIEW

File: src/protocols/rdp/settings.c [hunk 1]
```diff
@@ -1393,6 +1393,11 @@ void guac_rdp_push_settings(guac_client* client,
     rdp_settings->AlternateShell = guac_strdup(guac_settings->initial_program);
     rdp_settings->KeyboardLayout = guac_settings->server_layout->freerdp_keyboard_layout;
 
+    /* Add GUACAMOLE-1503: Fixed so that full-width and half-width keys work correctly and Japanese input switching works. */
+    rdp_settings->KeyboardType = 7;
+    rdp_settings->KeyboardSubType = 2;
+    rdp_settings->KeyboardFunctionKey = 12;
```
Comment (mike-jumper) [inline src/protocols/rdp/settings.c:1399]: Fixed ... how? What is the reasoning behind this change, and what are these values? Are these values correct for all keyboards?

File: src/protocols/rdp/keymaps/ja_jp_qwerty.keymap [hunk 2]
```diff
@@ -32,5 +32,4 @@ map +shift      0x1E..0x28 0x2B ~ "ASDFGHJKL+*}"
 map +shift      0x2C..0x35 0x73 ~ "ZXCVBNM<>?_"
 
 map -shift      0x29            ~ 0xFF28
-map -shift      0x29            ~ 0xFF2A
-map +shift      0x29            ~ 0xFF29
+map -shift      0x29            ~ 0xFF29
```
Comment (mike-jumper) [inline src/protocols/rdp/keymaps/ja_jp_qwerty.keymap:35]: Can you clarify why this change is correct vs. the previous values?

**Comment by necouchman** [comment]
Seems like these changes might be related to #256 - maybe GUACAMOLE-1503 is a duplicate of GUACAMOLE-951?

https://issues.apache.org/jira/browse/GUACAMOLE-951

**Comment by nw-engineer** [comment]
I'm really sorry. I wasn't able to see the exchanges of past issues.（GUACAMOLE-520,GUACAMOLE-951)

Also, some change requests are mistaken.
I would like to confirm the contents again and send a separate pull request.

I canceled this pull est
What should I do now?

**Comment by nw-engineer** [comment]
I didn't comment in particular, so I will close it. I apologize to you for a fuss over.

**Comment by necouchman** [comment]
@nw-engineer: I was just commenting on that so that @mike-jumper could see that there was some possible work, already, on this issue, that might answer some of his questions. The other pull request that I mentioned has been inactive for quite a while, so if you're willing to continue to work on this pull request, it would help to close out multiple issues.
